test_lasso.sh

```diff
@@ -14,5 +14,5 @@ echo "First running mod_auth_mellon tests"
 
 if [ $MAJOR_VER -ne 8 ]; then
```

Comment:
I've added a check to python test itself to expect a failure on older versions of RHEL. So maybe this also can be deleted.

Reply:
I'm ok with deleting considering the other check you added for xfail but, would that confuse things to run the new test which will show xfail when it hasn't been run before (for RHEL8 for instance)?

Reply:
I can change to skipif instead of xfail. But both xfail and skipif tests show up as Waiting in the test run in Polarion. Or we can add a selector to not execute tests at all on the unsupported platforms.

Reply:
Let's leave xfail.  It clearly indicates the reason and where it's supported.   And I think I'm ok with removing this check.  If it causes confusion later, we can always add it back.